include/input.hpp

```diff
@@ -209,6 +209,7 @@ class VSStlsInput : public StlsInput {
   void print() const;
   // Compare two VSStls objects
   bool isEqual( const VSStlsInput &in ) const;
+  bool ThetaDeriv;

```

Comment:
Remove this, this belongs to a different pull request. In general one pull request should include only one set of changes and adding the option to include or not the Theta derivatives is not part of the ESA scheme